.github/workflows/cicd-main.yml

```diff
@@ -5450,6 +5450,21 @@ jobs:
       SCRIPT: |
         bash tests/collections/llm/bitexact/mixtral/run.sh
 
+  L2_NeMo_2_PTQ_Llama2_FP8:
+    needs: [cicd-test-container-setup]
+    uses: ./.github/workflows/_test_template.yml
+    if: contains(fromJSON(needs.cicd-test-container-setup.outputs.test_to_run), 'L2_NeMo_2_PTQ_Llama2_FP8') || needs.cicd-test-container-setup.outputs.all == 'true'
+    with:
+      RUNNER: self-hosted-azure
+      SCRIPT: |
+        python tests/collections/llm/test_hf_import.py --hf_model /home/TestData/nlp/megatron_llama/llama-ci-hf --output_path /tmp/nemo2_ckpt
+
+        python examples/llm/quantization/ptq.py -nc /tmp/nemo2_ckpt -algo fp8 -out /tmp/nemo2_ptq_engine
```

Comment:
Could we possibly use a tiny random dataset here? This can be used https://github.com/NVIDIA/NeMo/blob/main/tests/setup/data/create_sample_jsonl.py.

I'm not sure if in GH CI/CD system we are able to pull the large dataset using `from datasets import load_dataset` here https://github.com/NVIDIA/NeMo/blob/c5a2a4d0269afac34405ad59567c67d8c96a6ad3/nemo/collections/llm/quantization/quantizer.py#L313-L331. cc @ko3n1g

Reply:
This will be enabled in a subsequent MR as this tiny dataset results in `np.nan` / `np.inf` stats currently and can't be used